### PR TITLE
Port plugins to CNI 1.0.0 and increase old verison test coverage

### PR DESCRIPTION
**Pull request by dcbw**

### CODE REVIEW

File: test_linux.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ else
     echo "without coverage profile generation..."
 fi
 
-PKG=${PKG:-$(cd ${GOPATH}/src/${REPO_PATH}; go list ./... | xargs echo)}
+PKG=${PKG:-$(cd ${GOPATH}/src/${REPO_PATH}; go list ./... | grep -v dhcp | xargs echo)}
```
Comment (squeed) [inline test_linux.sh:29]: what's this.......?

Comment (dcbw) [reply test_linux.sh]: leftover, removed

**Review by bboreham — APPROVED** [review]
seems legit

**Comment by dcbw** [comment]
We agreed that each plugin must at least test that 1.0.0 works for ADD.

**Comment by dcbw** [comment]
@squeed @bboreham @jellonek @mccv1r0 I ran out of steam doing the "meta" plugins, got as far as flannel. What do you think now?

**Review by bboreham — APPROVED** [review]
Wow!

**Pull request merged**